react/Minicart.tsx

```diff
@@ -18,6 +19,7 @@ interface MinicartProps {
   openOnHover: boolean
   linkVariationUrl: string
   maxDrawerWidth: number | string
+  MinicartIcon: React.ComponentType
```

Comment:
How about moving this prop to `MinicartContext`? Since this is being passed around to multiple components.

Reply:
It seems a bit weird putting this in the context because no one else will use it and if we put it there, it would be part a "public API" of the minicart

Reply:
🤔 But the `MinicartContext` is only used inside of this app

Reply:
I talked about it and we decided to keep this way